Source/Blazorise/Components/ModalProvider/ModalProvider.razor

```diff
@@ -0,0 +1,39 @@
+﻿@namespace Blazorise
+@inherits BaseComponent
+@using Blazorise.Extensions
+
+<CascadingValue Value="@this" IsFixed>
+    @if (!modalInstances.IsNullOrEmpty())
+    {
+        @foreach (var modal in modalInstances)
+        {
+            <Modal @ref="modal.ModalRef" @bind-Visible="modal.Visible" Attributes="modal.Attributes" Animated="modal.Animated()" AnimationDuration="modal.AnimationDuration()" Background="modal.Background" Border="modal.Border" Casing="modal.Casing" Class="modal.Class"
```

Comment:
Add a `@key` for optimization.

Reply:
Right... damn memory... there should be some kinda of intelisense for that once you use a foreach haha

Reply:
Well, ask for the feature on the DotNetCore repo. Maybe they will listen 😆